code/unclassified/src/leap_year/Leap_Year.java

```diff
@@ -0,0 +1,44 @@
+import java.util.*;
+ 
+public class Leap_Year {
+
+	int year1, year2;
+	Scanner sc;
+	// getting value of the years from user
+	void getVal() {

```

Comment:
That isn't really a method of this class, year1 and year2 should be read outsite the class and passed through a constructor.